ironfish/src/account/accounts.ts

```diff
@@ -746,15 +753,30 @@ export class Accounts {
       // TODO: This algorithm suffers a deanonim attack where you can watch to see what transactions node continously
```

Comment:
deanonim == deanonymization?

Reply:
Typo is unrelated to this PR, but I'll just fix it here this time.